packages/protocol/thread/thread.ts

```diff
@@ -118,6 +117,12 @@ declare global {
   }
 }
 
+// Experimental feature
+let repaintAfterEvaluations: boolean = false;
+export function setRepaintAfterEvaluations(value: boolean): void {
+  repaintAfterEvaluations = value;
+}
```

Comment:
This pattern is a tiny bit awkward I guess but at least it doesn't leak references to outside packages?